index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Awsome Books</title>
+  </head>
+  <body>
+    <main>
+      <h1 id="main">Awesome Books</h1>
+      <ul class="book-list">
+        <!-- <div class="book-name">Book Name</div>
+        <div class="author-name">Author Name</div>
+        <button type="submit">Remove</button> -->
```

Comment:
- [ ] Please remove commented code as it makes your code difficult to read. :pray: